### PR TITLE
More logging = more knowledge of what is happening

### PR DESCRIPTION
**Pull request by jsuchome**

### CODE REVIEW

File: pkg/kubernetes/cluster.go [hunk 1]
```diff
@@ -74,20 +78,10 @@ func (c *Cluster) DeleteResource(ctx context.Context, name, namespace, kind stri
...
-		return fmt.Errorf("failed fetching the resource: %w", err)
-	}
-
 	err = dr.Delete(ctx, name, metav1.DeleteOptions{})
-	if err != nil {
+	if !k8serr.IsNotFound(err) {
 		return err
 	}
-
+	c.logger.Print("resource not found, no need to delete")
```
Comment (flaviodsr) [inline pkg/kubernetes/cluster.go:85]: Shouldn't you also mention which resource is this about? I see you mention it in a log before, but assuming 2+ delete operations at the same time will be difficult to find out which one this line is referring to.

**Review by flaviodsr — CHANGES_REQUESTED** [review]
Looks good, just a question.

File: pkg/kubernetes/cluster.go (hunk 1 above)
Comment (jsuchome) [reply pkg/kubernetes/cluster.go:85]: Right, let me fix it.

Comment (jsuchome) [reply pkg/kubernetes/cluster.go:85]: Ah, no. I just looked at it and it should not be necessary when deleting of each resource starts with 
`want to delete resource %s of kind %s in %s namespace"`

So this message ("resource not found") should occur right after the previous one specifying the resource name and kind.

**Review by flaviodsr — APPROVED** [review]

**Pull request merged**